Week2/hw-part1.js

```diff
@@ -0,0 +1,137 @@
+const prompts = require('prompts');
+const mysql = require('mysql');
+const util = require('util');
+const colors = require('colors');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+  multipleStatements: false,
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+const questions = [
+  {
+    type: 'text',
+    name: 'CountryCode',
+    message: 'Q1. Enter a country code to see its capital',
+    initial: 'Example: NLD',
+  },
+  {
+    type: 'text',
+    name: 'region',
+    message: 'Q2. Enter a region to find the languages',
+    initial: 'Example: South America',
+  },
+  {
+    type: 'text',
+    name: 'language',
+    message: 'Q3. Enter a language to find the number of cities it is spoken',
+    initial: 'Example: English',
+  },
+  {
+    type: 'text',
+    name: 'region2',
+    message: 'Q4. Enter a region ',
+    initial: 'Example: Middle East',
+  },
+  {
+    type: 'text',
+    name: 'language2',
+    message: 'Q4. Enter a language',
+    initial: 'Example: Arabic',
+  },
+  {
+    type: 'text',
+    name: 'continent',
+    message: 'To get continents with number of languages spoken',
+    initial: 'Just press enter',
+  },
+];
+
+async function queryDatabase() {
+  let inputCountryCode = '';
+  let inputRegion = '';
+  let inputLanguage = '';
+  let inputRegOf = '';
+  let inputLangOf = '';
+
+  try {
+    const onSubmit = (prompt, response) => console.log(`You typed ${colors.rainbow(response)}`);
+    const response = await prompts(questions, { onSubmit });
+
+    inputCountryCode = response.CountryCode.toUpperCase();
+    inputRegion = response.region
```

Comment:
Did you have problem with the Case in your DB ? usually by default it's case insensitive so you don't have to pay attention to upper case at the beginning of input word. But otherwise the code is good

Reply:
No I only did it for my eye's pleasure:D But you are right it's unnecessary I removed it :)